lib/components/modal.vue

```diff
@@ -104,6 +104,22 @@
         '[tabindex]:not([disabled]):not(.disabled)'
     ].join(',');
 
+    // to handle multiple modal
+    var modalGlobalCounter = {
```

Comment:
Note in Server Side Rendering (SSR), this global variable is shared between ALL requests.  So if a modal is set to be initially show from server side, this will affect all requests to every user of the website.